Plugins/Phoenix/KalturaPluginManager.swift

```diff
@@ -136,8 +145,22 @@ final class KalturaPluginManager {
             t.invalidate()
         }
     }
+}
+
+/************************************************************/
+// MARK: - App State Handling
+/************************************************************/
+
+extension KalturaPluginManager: AppStateObservable {
```

Comment:
this is the reason of this bug :)
we need to have stop analytics event on terminate.